docs/content/execute.md

```diff
@@ -79,6 +79,10 @@ execute:
 
 Any file that matches one of the items in `exclude_patterns` will not be executed.
 
+If you would like to disable all files outside of your toc by default, use `only_build_toc_files: true` in your `_config.yml`.
```

Comment:
We should put this somewhere other than `execute`, as this configuration doesn't strictly have anything to do with execution, it has to do with building the files in the first place (and thus, execution as a secondary step). I'm not sure the best place, but maybe a start is here: https://jupyterbook.org/customize/config.html with cross-refs from the execute page as well as the `build` page?

Reply:
also we should add a header above this information so that it's more discoverable. I find that embedding configuration values within sentences makes it much harder for people to quickly find. Better to use patterns like:

````
**To disable building any files that aren't specified in toc**, use the following pattern in `_config.yml`:

```
only_build_toc_files: true
```
````

etc.

Reply:
Thanks for the tips. I'm not so great in the docs department but hopefully this looks good now -- just let me know.